t/05-countries.t

```diff
@@ -0,0 +1,87 @@
+#!perl
+use strict;
+use Test::More tests => 9;
+use Data::Dumper;
+
+use Net::Easypost;
+use Net::Easypost::Address;
+use Net::Easypost::Parcel;
+use Net::Easypost::Rate;
+
+$ENV{EASYPOST_API_KEY} = 'Ao0vbSp2P0cbEhQd8HjEZQ';
+
+if (!eval { require Socket; Socket::inet_aton('www.easypost.com') }) {
+    plan skip_all => "Cannot connect to the API server";
+}
+
+# 60 second connection timeout
+$ENV{MOJO_CONNECT_TIMEOUT} = 60;
+
+my $easypost = Net::Easypost->new( access_code => 'Ao0vbSp2P0cbEhQd8HjEZQ' );
+
+my @us_rates = get_rates(US => 22902 => 100);
+my @it_rates = get_rates(IT => 22902 => 100);
```

Comment:
This would be more idiomatically written as 

```perl
my @us_rates = get_rates('US', 22902, 100);
my @it_rates = get_rates('IT', 22902, 100);
```